doc/rst/developer/bestPractices/GeneratedCode.rst

```diff
@@ -29,7 +29,7 @@ Why are my identifiers renamed in the generated code?
   from other C libraries and headers against which Chapel is linked
   (see the ``chpl`` man page for ``--[no-]munge-user-idents`` for more
   information).  This can have the downside of making the generated
-  code not correspond with the Chapel source in a way that can make
+  code does not correspond with the Chapel source in a way that can make
```

Comment:
This is incorrect: "of making the generated code does not" is not a proper portion of a sentence.  You could replace the "not" with "fail to" (e.g. "This can have the downside of making the generated code fail to correspond with ... "), but "not" is sufficient.